packages/docsite/src/pages/docs/learn/getting_started.md

```diff
@@ -0,0 +1,314 @@
+  hotspots: Asset[],
+): Promise<HotspotWithPendingRewards[]> {
+  const program = await lz.init(provider)
+  const hemProgram = await init(provider)
+  const dao = DAO_KEY
+  const keyToAssets = hotspots.map((h) =>
+    keyToAssetForAsset(h),
+  )
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const ktaAccs = await getCachedKeyToAssets(hemProgram as any, keyToAssets)
+  const entityKeys = ktaAccs.map(
+    (kta) =>
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      decodeEntityKey(kta.entityKey, kta.keySerialization)!,
+  )
+
+  const mobileRewards = await getPendingRewards(
+    program,
+    MOBILE_LAZY_KEY,
+    dao,
+    entityKeys,
+  )
+
+  const iotRewards = await getPendingRewards(
+    program,
+    IOT_LAZY_KEY,
+    dao,
+    entityKeys,
+  )
+
+  return hotspots.map((hotspot, index) => {
+    const entityKey = entityKeys[index]
+
+    return {
+      ...hotspot,
+      pendingRewards: {
+        [Mints.MOBILE]: mobileRewards[entityKey],
+        [Mints.IOT]: iotRewards[entityKey],
+      },
+    } as HotspotWithPendingRewards
+  })
+}
+```
+
+### Generate claim txns and send txns in batches
+
+```js async name=generate-claim-txns
+import { AnchorProvider } from '@coral-xyz/anchor'
+import { Asset, HNT_MINT } from '@helium/spl-utils'
+import * as lz from '@helium/lazy-distributor-sdk'
+import { init, keyToAssetForAsset, decodeEntityKey } from '@helium/helium-entity-manager-sdk'
+import { daoKey } from '@helium/helium-sub-daos-sdk'
+import { getPendingRewards, formBulkTransactions, getBulkRewards } from '@helium/distributor-oracle'
+
+export const claimAllRewards = 
+  async (
+    {
+      account,
+      anchorProvider,
+      cluster,
+      lazyDistributors,
+      hotspots,
+    } : {
+      account: string,
+      anchorProvider: AnchorProvider,
+      cluster: string,
+      lazyDistributors: PublicKey[],
+      hotspots: HotspotWithPendingRewards[],
+    }
+  ) => {
+    try {
+      const ret: string[] = []
+      let triesRemaining = 10
+      const program = await lz.init(anchorProvider)
+      const hemProgram = await init(anchorProvider)
+
+      const mints = await Promise.all(
+        lazyDistributors.map(async (d) => {
+          return (await program.account.lazyDistributorV0.fetch(d)).rewardsMint
+        }),
+      )
+      const ldToMint = lazyDistributors.reduce((acc, ld, index) => {
+        acc[ld.toBase58()] = mints[index]
+        return acc
+      }, {} as Record<string, PublicKey>)
+      // One tx per hotspot per mint/lazy dist
+      const totalTxns = hotspots.reduce((acc, hotspot) => {
+        mints.forEach((mint) => {
+          if (
+            hotspot.pendingRewards &&
+            hotspot.pendingRewards[mint.toString()] &&
+            new BN(hotspot.pendingRewards[mint.toString()]).gt(new BN(0))
+          )
+            acc += 1
+        })
+        return acc
+      }, 0)
+      dispatch(
+        solanaSlice.actions.setPaymentProgress({
```

Comment:
Needs better example code. This is mixed in with nasty redux logic.

Reply:
There was only one set of leftover redux logic. Its been removed